tark/tark/settings/env.py

```diff
@@ -19,5 +19,5 @@
 
 ENV = os.getenv('DJANGO_ENVIRONMENT', 'development')
```

Comment:
For the sake of consistency, I think this should be either `dev` or `development` based on what we decide to use here (in [tark-configs repo](https://github.com/Ensembl/tark-configs)):
```
# Replace the following with 'dev' or 'staging' or 'primary'
Define PROJECT_ENV REPLACE_ME
```

Reply:
Updated in config repo